src/types/commonTypes.ts

```diff
@@ -3,7 +3,7 @@ export enum SupportedLanguages {
   zh = 'zh',
 }
 
-export enum AuctionStatus {
+export enum LotAuctionStatus {
```

Comment:
We will probably need two separate enums here. I assumed the lot status check would have been done in the remix project. If we are adding an additional enum here we need to still support AuctionStatus.